docs/content/functioneel/zaken_api.md

```diff
@@ -0,0 +1,104 @@
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een zaak toevoegen, gegevens van een zaak wijzigen, wijzigingen aanbrengen in de relatie van een zaak tot betrokkenen en/of objecten dan wel één of meer gegevens van zaken en genoemde relaties opvragen.                                                                                                            |
+| Voorwaarde     | 1. Het zaaktype van de zaak moet al bestaan en geldig zijn (resource: zaaktypen in API: ZTC). <br>2.  Een te relateren Betrokkene moet al bestaan (resource: [...] in API:     [...]). <br>3.  Een te relateren Object moet al bestaan (resource: [...] in API: [...]). <br>4.  Bij het toevoegen van een zaak mag de (fubctionele) (Zaak)identificatie meegegeven worden; deze dient uniek te zijn. |
+| Gevolg         | De zaak wordt, indien sprake is van valide gegevens, toegevoegd dan wel gewijzigd door de Zaakregistratiecomponent dan wel worden gegevens van de gevonden zaken beschikbaar gesteld. Indien t.b.v. het toevoegen van een zaak geen (Zaak)identificatie is meegeleverd dan wordt deze door de ZRC bepaald. |
+| Gegevens       | Objecttype ZAAK, inclusief relaties naar Zaaktype, Betrokkene (incl. relatieklasse ROL) en OBJECT (incl. relatieklasse ZAAKOBJECT). <br>Zie de volgende tabel voor de gegevens van deze resource |
+| Specificaties  | <https://ref.tst.vng.cloud/zrc/api/v1/schema/#tag/zaken>                                                                                                                                                                                                                                                                               |
+| Samenhang      | Update van Zaak.Einddatum vindt plaats vanuit de resource Statussen.                                                                                                                                                                                                                                                                   |
+| Bijzonderheden | -  De operatie DELETE wordt niet ondersteund: zaken kunnen niet verwijderd worden. <br>-   Een zaak moet tenminste één betrokkene in een Rol hebben. <br>-   De relatie naar Zaaktype (cq. het zaaktype van de zaak) kan niet gewijzigd worden. Indien het zaaktype onjuist blijkt te zijn, dient de zaak beeindigd te worden en een nieuwe zaak (met het juiste zaakype) toegevoegd te worden. <br>-   De Einddatum van de zaak kan alleen opgevraagd worden. Muteren hiervan gebeurt in de resource: Statussen. |
+
+
+*Gegevens*
+
+[tabel van Sergei met de gegevens van de resource Zaken]
```

Comment:
ik *denk* dat we dit met een markdown include kunnen doen straks, want die tabel wordt natuurlijk ook gegenereerd.

Reply:
@sergei-maertens  Ik reken er op. Kun je per resource dwz. per tabel een apart markup-document maken? In een map binnen de map 'functioneel' (waarin de functionele documentatie staat)? Dan staat het mooi bij elkaar.
En kun je in die tabel per resource-element ook de corresponderede RGBZ-attribuut- of -relatienaam vermelden?

Reply:
@ArjanKloosterboer net even gezocht en op het eerste zicht lijkt het niet mogelijk om uberhaupt includes te doen. Misschien dat we met de 'nieuwe' documentatietooling wel wat kunnen, maar ik ga er niet van uit. Wat wel makkelijk is, is linken naar het document die deze resources bevat.

Een apart markup doc per resource kan wel, alleen zie ik niet meteen de meerwaarde ten opzichte van 1 enkel document met eventueel een terms-of-content?

> En kun je in die tabel per resource-element ook de corresponderede RGBZ-attribuut- of -relatienaam vermelden?

Nee, die hebben we ook niet in de API spec als dusdanig.